lib/resources/dstu2/patient.yaml

```diff
@@ -0,0 +1,219 @@
+---
+name: Patient
+field_name_base_url: http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient
+fields:
+- name: identifier
+  required: 'No'
+  cardinality: 0..*
+  type: Identifier
+  description: The identifier for the patient.
+  children:
+
+  - name: use
+    required: 'No'
+    cardinality: 0..1
+    type: code
+    description: The purpose of the identifier.
+    binding:
```

Comment:
Removed the non required bindings. We only need to document Patient.identifier.type and Patient.contact.relationship.